README.md

```diff
@@ -1 +1,28 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# mobile-version
+
+> >It is a project of the microverse which consists in evaluating and boosting the knowledge of their learners in the course of Github although you can notice there HTML and CSS. this course has a capital utility because you will understand how to make pull request, how to merge on branches also how to write clear code with linters.
+
+Additional description about the project and its features.
+
+## Built With
+
+- Html
+- Css
+- Github
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+## Authors
+
+👤 **Kevin Koome**
+
+- GitHub: [@githubhandle](https://github.com/koomekevo)
+- Twitter: [@twitterhandle](https://twitter.com/koomekevo)
+
+/in/linked
```

Comment:
- [ ] Please kindly remove this as it make your readme dirty. Visitors are potential recruiters need to see that you page attention to details.